apps/api/src/auth/login.controller.ts

```diff
@@ -15,4 +17,15 @@ export class LoginController {
   async login(@Body() loginDto: LoginDto) {
     return await this.authService.login(loginDto)
   }
+
+  @Post('/forgot-password')
+  @Public()
+  async forgotPassword(@Body() forgotPasswordDto: ForgotPass) {
+    return await this.authService.forgotPass(forgotPasswordDto)
+  }
+  @Post('/recovery-password')
+  @Public()
+  async recoveryPassword(@Body() RecoveryPasswordDto: RecoveryPasswordDto) {
+    return await this.authService.recoveryPass(RecoveryPasswordDto)
+  }
```

Comment:
let's rename to reset-password, because recovery is not suitable for this activity